app/routes/core.rb

```diff
@@ -21,7 +21,7 @@ class Core < Sinatra::Application
         enable :show_exceptions
       end
 
-      unless settings.test?
+      unless settings.test? || settings.development?
```

Comment:
This line is redundant.

Whether detailed Sinatra exceptions are displayed is controlled by `:show_exceptions` on line 21 above. 

I would suggest removing this conditional completely.